roles/translator/src/lib/downstream_sv1/downstream.rs

```diff
@@ -242,11 +251,18 @@ impl Downstream {
                 &host_
             );
         });
+        let _ = task_collector_new_sv1_message_no_transl.safe_lock(|a| {
+            a.push((
+                socket_writer_task.abort_handle(),
+                "socket_writer_task".to_string(),
+            ))
```

Comment:
Any specific reason adding context to abort handle?

Reply:
it seems the purpose of these strings is so that `kill_tasks` can print them to the console, like this:
```
2024-08-01T23:36:40.372202Z  WARN translator_sv2: Killed task: "notify_task"
2024-08-01T23:36:40.372229Z  WARN translator_sv2: Killed task: "socket_writer_task"
2024-08-01T23:36:40.372233Z  WARN translator_sv2: Killed task: "socket_reader_task"
2024-08-01T23:36:40.372237Z  WARN translator_sv2: Killed task: "accept_connections"
2024-08-01T23:36:40.372240Z  WARN translator_sv2: Killed task: "handle_downstream_message"
2024-08-01T23:36:40.372244Z  WARN translator_sv2: Killed task: "handle_new_extended_mining_job"
2024-08-01T23:36:40.372247Z  WARN translator_sv2: Killed task: "handle_new_prev_hash"
2024-08-01T23:36:40.372251Z  WARN translator_sv2: Killed task: "handle_submit"
2024-08-01T23:36:40.372258Z  WARN translator_sv2: Killed task: "parse_incoming"
2024-08-01T23:36:40.372360Z  WARN translator_sv2: Killed task: "start_diff_management"
2024-08-01T23:36:40.372365Z  WARN translator_sv2: Killed task: "init task"
```